sdk/azidentity/go.mod

```diff
@@ -2,11 +2,9 @@ module github.com/Azure/azure-sdk-for-go/sdk/azidentity
 
 go 1.18
 
-replace github.com/Azure/azure-sdk-for-go/sdk/azcore => ../azcore
-
 require (
-	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.23.0
-	github.com/Azure/azure-sdk-for-go/sdk/internal v0.9.1
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.0.0
+	github.com/Azure/azure-sdk-for-go/sdk/internal v1.0.0
 	github.com/AzureAD/microsoft-authentication-library-for-go v0.4.0
 	github.com/golang-jwt/jwt/v4 v4.2.0
 	golang.org/x/crypto v0.0.0-20201016220609-9e8e0b390897
```

Comment:
Can you check if there's a newer version of this dependency?  `go get -u golang.org/x/crypto`